### PR TITLE
Add support for multiple sql load scripts

### LINKED ISSUE #20177: Multiple files in sql-load-script
### Describe the bug

Cannot use `quarkus.hibernate-orm.sql-load-script` to specify a list of files.

### Expected behavior

As #9464 is closed, specifying multiple import.sql files should be supported.

### Actual behavior

An exception is thrown:

```
java.lang.RuntimeException: java.lang.RuntimeException: io.quarkus.builder.BuildException: Build failure: Build failed due to errors
	[error]: Build step io.quarkus.hibernate.orm.deployment.HibernateOrmProcessor#configurationDescriptorBuilding threw an exception: io.quarkus.deployment.configuration.ConfigurationError: Unable to find file referenced in 'quarkus.hibernate-orm.sql-load-script=import-1.sql,import-2.sql'. Remove property or add file to your path.

```

### How to Reproduce?

1. Take hibernate-orm-quickstart
2. Add two files `import-1.sql` and `import-2.sql`
3. Edit `application.properties` and change the following config with the comma-separated list of the two files:

    ```
    quarkus.hibernate-orm.sql-load-script=import-1.sql,import-2.sql
    ```

### Output of `uname -a` or `ver`

_No response_

### Output of `java -version`

_No response_

### GraalVM version (if different from Java)

_No response_

### Quarkus version or git rev

2.2.2.Final

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

Maven 3.8.1

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by mschnitzler**
This PR fixes the issue #20177.
The property "quarkus.hibernate-orm.sql-load-script" can now take a comma separated list of SQL load scripts.

### CODE REVIEW

File: extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmProcessor.java [hunk 1]
```diff
@@ -1021,26 +1023,19 @@ private static void producePersistenceUnitDescriptorFromConfig(
             descriptor.getProperties().setProperty(AvailableSettings.GENERATE_STATISTICS, "true");
         }
 
-        // sql-load-script
+        // sql-load-scripts
         Optional<String> importFile = getSqlLoadScript(persistenceUnitConfig.sqlLoadScript, launchMode);
 
         if (importFile.isPresent()) {
-            Path loadScriptPath = applicationArchivesBuildItem.getRootArchive().getChildPath(importFile.get());
+            Boolean allLoadScriptFilesExist = Stream.of(importFile.get().split(","))
```
Comment (yrodiere) [inline extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmProcessor.java:1030]: You don't need to do the splitting yourself; I think you can just change the type of the `sqlLoadScript` property to `Option<List<String>>`. That would be more explicit, I believe?

File: extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmProcessor.java [hunk 2]
```diff
@@ -1021,26 +1023,19 @@ private static void producePersistenceUnitDescriptorFromConfig(
...
+                    .map(String::trim)
+                    .map(handleLoadScriptFile(persistenceUnitName, persistenceUnitConfig, applicationArchivesBuildItem,
```
Comment (yrodiere) [inline extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmProcessor.java:1032]: I'll admit it's nitpicking, but... can't you just use a loop? That would be clearer IMO. You can still call another method from that loop, if you want to.

One thing that is definitely not clear with this code is that this stream will only ever reduce to `false` if `sqlLoadScript` defaulted to `import.sql` and that file does not exist. If an explicitly configured file does not exist, an exception will be thrown instead.

**Review by yrodiere — COMMENTED** [review]
Great to fix that, thanks!

I have a few comments, but nothing blocking. Let me know what you think?

File: extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmProcessor.java (hunk 2 above)
Comment (mschnitzler) [reply extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmProcessor.java]: I agree, a loop is probably easier to read here.
Thanks for your feedback. I submitted another commit with your suggestions.

**Review by yrodiere — APPROVED** [review]
Looks great, thanks.

**Pull request merged**